src/main/java/com/ericsson/ei/subscriptionhandler/RunSubscription.java

```diff
@@ -107,19 +107,23 @@ public boolean runSubscriptionOnObject(String aggregatedObject, Iterator<JsonNod
             while (conditionIterator.hasNext()) {
                 String rule = conditionIterator.next().get("jmespath").toString().replaceAll("^\"|\"$", "");
                 String new_Rule = rule.replace("'", "\"");
-                LOGGER.info("Rule : " + rule);
-                LOGGER.info("New Rule after replacing single quote : " + new_Rule);
+                LOGGER.debug("New Rule after replacing single quote : " + new_Rule);
                 JsonNode result = jmespath.runRuleOnEvent(rule, aggregatedObject);
-                LOGGER.info("Result : " + result.toString());
-                if (result.toString() != null && result.toString() != "false" && !result.toString().equals("[]")) {
+                boolean condition1 = !result.toString().equals("null");
```

Comment:
What are you checking here? Is the extracted content?
Rename also the conditionX varables to something meaningful, e.g : isContentNull", "isContentFalse", etc,